libs/platform/src/lib/components/form/checkbox-group/checkbox-group.component.ts

```diff
@@ -65,6 +77,13 @@ export class CheckboxGroupComponent extends CollectionBaseInput {
     @Output()
     readonly valueChange: EventEmitter<PlatformCheckboxChange> = new EventEmitter<PlatformCheckboxChange>();
 
+    /** Emits checked change event */
+    @Output()
+    readonly checkedChange: EventEmitter<String[]> = new EventEmitter<String[]>();
+
```

Comment:
Why do we need the `checkedChange` EventEmitter when we already have the `valueChange` EventEmitter? Don't they do the same thing?

Reply:
On second thought, I guess this is to enable two-way binding.

Reply:
`String[]` should be lower case -> `string[]`

Reply:
> Why do we need the `checkedChange` EventEmitter when we already have the `valueChange` EventEmitter? Don't they do the same thing?

In valueChanged, we have passed an object which has a property which denotes which checkbox was changed and the final values of checked checkboxes. 

Here `value` is not `Input` for two way binding we need only checked checkboxes value. 
So for avoiding any breaking change, we need to introduce `checkedChange` property.